src/main/java/frc/team2412/robot/commands/intake/IntakeBitmapCommand.java

```diff
@@ -0,0 +1,98 @@
+package frc.team2412.robot.commands.intake;
+
+import static frc.team2412.robot.subsystem.IntakeSubsystem.IntakeConstants.*;
+import static frc.team2412.robot.subsystem.IndexSubsystem.IndexConstants.*;
+import static frc.team2412.robot.subsystem.ShooterSubsystem.ShooterConstants;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.IndexSubsystem;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+import frc.team2412.robot.subsystem.ShooterSubsystem;
+import frc.team2412.robot.subsystem.ShooterVisionSubsystem;
+import frc.team2412.robot.util.ShooterDataDistancePoint;
+
+public class IntakeBitmapCommand extends CommandBase {
+
+    public final double MISFIRE_VELOCITY = 400;;
+
+    public enum Bitmap {
+        // ingesthasball, feederhasball, ingestcorrectcolor, feedercorrectcolor, intakespeed, ingestspeed,
+        // feederspeed, misfire
+        A(false, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false), // no balls in systems
+        B(true, false, true, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false), // correct ball in ingest
+        C(true, false, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, false), // wrong ball in ingest
+        D(false, true, false, true, INTAKE_IN_SPEED, INDEX_IN_SPEED, 0, false), // correct ball in feeder
+        E(false, true, false, false, INTAKE_IN_SPEED, INDEX_IN_SPEED, INDEX_IN_SPEED, true), // wrong ball in feeder
+        F(true, true, true, true, 0, 0, 0, false), // correct ball in both
```

Comment:
I'm guessing the process of feeding + shooting is handled by another command? (maybe `IndexShootCommand`)
If so, how does that command know to end? (so we can use it in command groups)